schemas/styles/csl.rnc

```diff
@@ -1136,6 +1144,12 @@ div {
   always-render = attribute always-render { xsd:boolean }?
 }
 
+## Select a related-variable
+div {
+  relation = attribute relation { relations }?
```

Comment:
```suggestion
  related = attribute relation { relations }?
```

Reply:
I guess you want me to change the attribute name as well...

Reply:
No, everywhere else in CSL.rnc you refer to "related" but never define that variable. I don't care much what variable name is chosen. The attribute name is fine.

Reply:
Hmmm. I've thought I had fixed that...  maybe I'll need to update my branch

Reply:
Oh no. I've only changed it in csl-choose.rnc